pkg/kritis/review/validating_transport.go

```diff
@@ -101,51 +103,74 @@ func (avt *AttestorValidatingTransport) validatePublicKeyId(pubKey v1beta1.Publi
 	return nil
 }
 
+// NOTE: I am considering changing this function signature to:
```

Comment:
I advise against doing this.  In the bright future of attestations containing payloads, the caller of this function will be interested in the payload.

Reply:
Done.